### PR TITLE
feat: add possibility to assign additional CIDR blocks

### PR DESCRIPTION
**Pull request by SweetOps**
## what
* add possibility to assign additional CIDR blocks

## why
new feature

### CODE REVIEW

File: versions.tf [hunk 1]
```diff
@@ -1,10 +1,17 @@
 terraform {
-  required_version = ">= 0.12.0, < 0.14.0"
-
   required_providers {
-    aws      = ">= 2.0, < 4.0"
-    template = "~> 2.0"
-    local    = "~> 1.2"
-    null     = "~> 2.0"
+    aws = {
```
Comment (Gowiem) [inline versions.tf:3]: The new `required_providers` syntax was introduced for backwards compatibility in v0.12.26 -- Can you update the lower bound `required_version` to reflect that? Or don't update these at all and instead just remove the upper bound `required_version`.

**Review by Gowiem — CHANGES_REQUESTED** [review]
Looks good -- One request on the terraform.tf updates.

File: versions.tf (hunk 1 above)
Comment (SweetOps) [reply versions.tf:3]: pinned lower version to `v0.12.26`

**Review by Gowiem — APPROVED** [review]

**Pull request merged**